client/src/Components/Footer.jsx

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import { Link } from "react-router-dom";
+import { useTranslation } from "react-i18next";
+
+const Footer = () =>
+{
+    let currentYear = new Date().getFullYear();
+    const {t} = useTranslation();
+    const {ul1,ul2,ul3,ul4} = t('ulTranslation')
+
+    return(
+        <div className="bg-green-200 py-5 flex space-x-[1050px]">
```

Comment:
is there a way to visualize/validate parts of JSX in tool? Otherwise it would be hard to debug it

Reply:
I didn't get your point.
The code which you have highlighted here is:
`const {ul1,ul2,ul3,ul4} = t('ulTranslation')`
This will make ul1, ul2, ul3, ul4 to be translated as per selected user lang.

And this is: 

`<div className="bg-green-200 py-5 flex space-x-[1050px]">`
Core tailwind

Reply:
No need to rerun the full client again, code changes are applied live. Further action pending on testing